rahul/exercise12 - Prompting people for numbers/prompting-people-for-numbers-drill.rb

```diff
@@ -0,0 +1,5 @@
+print 'Give some  money: '
+get_money = gets.chomp.to_f
+
+pay_back = get_money * 0.1
```

Comment:
Do we really need a new variable here? 

Since we are simply printing the string, how about we do this:
`puts 'you will get 10 of the money back: %.2f' % get_money * 0.1`